### PR TITLE
*: export `advertise-addr` as the command arg; use `master-addr`

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

export `advertise-addr` to make DM easier to use in cloud.

### What is changed and how it works?

export `advertise-addr` as the command arg; use `master-addr`

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@WangXiangUSTC @lichunzhu PTAL

File: dm/master/config.go [hunk 1]
```diff
@@ -202,18 +203,25 @@ func (c *Config) configFromFile(path string) error {
...
 
-	// AdvertiseAddr's format must be "host:port"
-	host, port, err := net.SplitHostPort(c.AdvertiseAddr)
-	if err != nil {
-		return terror.ErrMasterAdvertiseAddrNotValid.Delegate(err, c.AdvertiseAddr)
-	}
-	if len(host) == 0 || len(port) == 0 {
-		return terror.ErrMasterAdvertiseAddrNotValid.Generate(c.AdvertiseAddr)
+	if c.AdvertiseAddr == "" {
+		if host == "" || host == "0.0.0.0" {
```
Comment (lichunzhu) [inline dm/master/config.go]: Should we check port here?

Comment (csuzhangxc) [reply dm/master/config.go]: addressed in https://github.com/pingcap/dm/pull/484/commits/e7b1f86e81423420748b3241c0cdcba3193b42f4.

**Pull request merged**